server/tests/common/data.py

```diff
@@ -211,3 +215,167 @@ async def _alice_generated_realm_data(now: DateTime) -> None:
+        tester_called = True
+        match request.param:
+            case "organization_expired":
+                outcome = await backend.organization.update(
+                    id=coolorg.organization_id, is_expired=True
+                )
+                assert outcome is None
+
+                expected_http_status = 460
+
+            case "author_revoked":
+                # Must first promote Bob to ADMIN...
+
+                bob_user_update_certificate = UserUpdateCertificate(
+                    author=coolorg.alice.device_id,
+                    timestamp=DateTime.now(),
+                    user_id=coolorg.bob.user_id,
+                    new_profile=UserProfile.ADMIN,
+                ).dump_and_sign(coolorg.alice.signing_key)
+
+                outcome = await backend.user.update_user(
+                    organization_id=coolorg.organization_id,
+                    now=DateTime.now(),
+                    author=coolorg.alice.device_id,
+                    author_verify_key=coolorg.alice.signing_key.verify_key,
+                    user_update_certificate=bob_user_update_certificate,
+                )
+                assert isinstance(outcome, UserUpdateCertificate)
+
+                # ...to be able to revoke Alice
+
+                certif = RevokedUserCertificate(
+                    author=coolorg.bob.device_id,
+                    user_id=coolorg.alice.user_id,
+                    timestamp=DateTime.now(),
+                )
+                outcome = await backend.user.revoke_user(
+                    now=DateTime.now(),
+                    organization_id=coolorg.organization_id,
+                    author=coolorg.bob.device_id,
+                    author_verify_key=coolorg.bob.signing_key.verify_key,
+                    revoked_user_certificate=certif.dump_and_sign(coolorg.alice.signing_key),
+                )
+                assert isinstance(outcome, RevokedUserCertificate)
+
+                expected_http_status = 461
+
+            case "author_frozen":
+                outcome = await backend.user.freeze_user(
+                    organization_id=coolorg.organization_id,
+                    user_id=coolorg.alice.user_id,
+                    user_email=None,
+                    frozen=True,
+                )
+                assert isinstance(outcome, UserInfo)
+
+                expected_http_status = 462
+
+            case unknown:
+                assert False, unknown
+
+        try:
+            await do()
+            assert False, f"{do!r} was expected to raise an `RpcTransportError` exception !"
+        except RpcTransportError as err:
+            assert err.rep.status_code == expected_http_status
+
+    yield _http_46x_authenticated_tester
+
+    assert tester_called
+
+
+# TODO: not sure how to test "organization_not_found" & "invitation_not_found"
+@pytest.fixture(params=("organization_expired", "invitation_already_used"))
+async def http_46x_invited_tester(
```

Comment:
I'd rename this functions (See comment below in `test_api.py`). Also I would put family first, I think is clearer when using it as fixture.

Something like:
```suggestion
async def invited_http_common_errors_tester(
```